### ISSUE TITLE
Allowinsecure goes back to NO 

### ISSUE THREAD

**Issue by essoojay** [v2rayA/v2rayA#264]
For Xray / vless and Trojan-Go, Allowinsecure always NO after changing to YES and saved. sometimes its needed to be YES ..is this a bug or its by design of program. 

thanks

**Comment by malikshi** [comment]
For Vless they're set to NO. VLESS SHOULD not support allowinsecure according to the protocol design spec. Other than that can be set allowinsecure Yes

**Comment by essoojay** [comment]
> For Vless they're set to NO. VLESS SHOULD not support allowinsecure according to the protocol design spec. Other than that can be set allowinsecure Yes

Thanks, i get the protocol design part.  on v2raya its an option the YES or NO i thought the Yes should work as well as the NO for users to choose. otherwise no need to give the YES option. Is there a particular way to make the YES option stick. Thanks

**Comment by malikshi** [comment]
>@essoojay Thanks, i get the protocol design part. on v2raya its an option the YES or NO i thought the Yes should work as well as the NO for users to choose. otherwise no need to give the YES option. Is there a particular way to make the YES option stick. Thanks

yup, need to add tooltips for the info about those particular settings would be nice or just allowing all types of services can be set YES/NO same as an android client(v2rayNG, Sagernet) did it

**Comment by mzz2017** [comment]
Thanks. I will give a prompt to explain it.
